internal/memdb/memdb_test.go

```diff
@@ -172,30 +171,3 @@ func TestOnWriteCtxCancelled(t *testing.T) {
 		t.Fatal(err)
 	}
 }
-
-func TestPluginInit(t *testing.T) {
```

Comment:
This was moved to https://github.com/cloudquery/plugin-sdk/pull/870/files#diff-f4ba5b8e6a3a7d42e936c4f99905060cbf7cace1ec07b3c284d6a2fda303dd8e